ReactNativeTemplate/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Mon Aug 22 10:53:08 PDT 2016
```

Comment:
Should we move these common files, i.e., `gradle-wrapper.jar`, `gradle-wrapper.properties`, `gradlew` to a common top level folder under `android` and symlink them here for each app template?